### PR TITLE
Added lint, build, and publish actions.

### PR DESCRIPTION
**Pull request by cshamrick**

### CODE REVIEW

File: sts_auth/okta.py [hunk 1]
```diff
@@ -25,41 +25,38 @@ def __init__(self, session, state_token, okta_org, okta_shared_secret=None):
...
         if len(okta_factors) > 0:
             # Format the factors as {factorType: {details..}, ..}
             # so it will be easier to pick pull out by type later
-            return {f['factorType']: f for f in okta_factors}
+            return {f["factorType"]: f for f in okta_factors}
         else:
-            msg = ('No Okta MFA methods available.\n'
-                   'Please visit https://{}.okta.com to configure Okta MFA.')
-            click.secho(msg.format(self.okta_org), fg='red')
+            msg = "No Okta MFA methods available.\n" "Please visit https://{}.okta.com to configure Okta MFA."
```
Comment (phillipjf) [inline sts_auth/okta.py:36]: Changes like these should be merged into one string if they're moved to a single.

File: sts_auth/okta.py [hunk 2]
```diff
@@ -25,41 +25,38 @@ def __init__(self, session, state_token, okta_org, okta_shared_secret=None):
...
     def handle_okta_verification(self, response):
         # If there is no assertion, and we find an Okta portal on the page,
         # we have to pass through the Okta portal regardless of whether MFA
         # will be required or not.
         if not self.okta_org:
-            msg = ('Okta MFA required but no Okta Organization set. '
-                   'Please either set in the config or use `--okta-org`')
-            click.secho(msg, fg='red')
+            msg = (
+                "Okta MFA required but no Okta Organization set. " "Please either set in the config or use `--okta-org`"
```
Comment (phillipjf) [inline sts_auth/okta.py:46]: Changes like these should be merged into one string if they're moved to a single.

File: sts_auth/okta.py [hunk 3]
```diff
@@ -93,79 +90,76 @@ def get_verification_status_from_response(self, response):
...
             if verify_response.ok:
-                status = verify_response.json().get('status')
-                if status == 'SUCCESS':
+                status = verify_response.json().get("status")
+                if status == "SUCCESS":
                     return True
         click.secho(
-            'TOTP Verification failed. '
-            'Continuing to other methods if available', fg='red'
+            "TOTP Verification failed. " "Continuing to other methods if available",
```
Comment (phillipjf) [inline sts_auth/okta.py:134]: Changes like these should be merged into one string if they're moved to a single.

File: sts_auth/stsauth.py [hunk 4]
```diff
@@ -93,39 +110,37 @@ def parse_config_file(self):
         """Read configuration file and only set values if they
         were not passed in from the CLI.
         """
-        if self.config.has_section('default'):
-            logger.debug('Found \'default\' section in'
-                         ' {0.credentialsfile!r}!'.format(self))
-            default = self.config['default']
-            msg = ('Attribute {1!r} not set, using value from {0.credentialsfile!r}')
+        if self.config.has_section("default"):
+            logger.debug("Found 'default' section in" " {0.credentialsfile!r}!".format(self))
```
Comment (phillipjf) [inline sts_auth/stsauth.py:114]: Changes like these should be merged into one string if they're moved to a single.

File: sts_auth/stsauth.py [hunk 5]
```diff
@@ -93,39 +110,37 @@ def parse_config_file(self):
...
+                self.okta_org = default.get("okta_org")
             if not self.okta_shared_secret:
-                logger.debug(msg.format(self, 'okta_shared_secret'))
-                self.okta_shared_secret = default.get('okta_shared_secret')
+                logger.debug(msg.format(self, "okta_shared_secret"))
+                self.okta_shared_secret = default.get("okta_shared_secret")
         else:
-            logger.debug('Could not find \'default\' section in'
-                         ' {0.credentialsfile!r}!'.format(self))
+            logger.debug("Could not find 'default' section in" " {0.credentialsfile!r}!".format(self))
```
Comment (phillipjf) [inline sts_auth/stsauth.py:136]: Changes like these should be merged into one string if they're moved to a single.

Comment (cshamrick) [reply sts_auth/stsauth.py]: @phillipjf fixed

File: sts_auth/stsauth.py (hunk 4 above)
Comment (cshamrick) [reply sts_auth/stsauth.py]: @phillipjf fixed

File: sts_auth/okta.py (hunk 3 above)
Comment (cshamrick) [reply sts_auth/okta.py]: @phillipjf fixed

File: sts_auth/okta.py (hunk 2 above)
Comment (cshamrick) [reply sts_auth/okta.py]: @phillipjf fixed

File: sts_auth/okta.py (hunk 1 above)
Comment (cshamrick) [reply sts_auth/okta.py]: @phillipjf fixed

**Review by phillipjf — APPROVED** [review]

**Review by phillipjf — APPROVED** [review]

**Pull request merged**